docs/api/non_geo_datasets.csv

```diff
@@ -1,51 +1,51 @@
+`Kenya Crop Type`_,S,Sentinel-2,"Apache-2.0","4,688",7,"3,035x2,016",10,MSI
+`DeepGlobe Land Cover`_,S,DigitalGlobe +Vivid,-,803,7,"2,448x2,448",0.5,RGB
+`DFC2022`_,S,Aerial,"CC-BY-4.0","3,981",15,"2,000x2,000",0.5,RGB
+`ETCI2021 Flood Detection`_,S,Sentinel-1,-,"66,810",2,256x256,5--20,SAR
+`EuroSAT`_,C,Sentinel-2,"MIT","27,000",10,64x64,10,MSI
+`FAIR1M`_,OD,Gaofen/Google Earth,"CC-BY-NC-SA-3.0","15,000",37,"1,024x1,024",0.3--0.8,RGB
+`FireRisk`_,C,NAIP Aerial,"CC-BY-NC-4.0","91,872",7,"320x320",1,RGB
+`Forest Damage`_,OD,Drone imagery,"CDLA-Permissive-1.0","1,543",4,"1,500x1,500",,RGB
+`GID-15`_,S,Gaofen-2,-,150,15,"6,800x7,200",3,RGB
+`IDTReeS`_,"OD,C",Aerial,"CC-BY-4.0",591,33,200x200,0.1--1,RGB
+`Inria Aerial Image Labeling`_,S,Aerial,-,360,2,"5,000x5,000",0.3,RGB
+`LandCover.ai`_,S,Aerial,"CC-BY-NC-SA-4.0","10,674",5,512x512,0.25--0.5,RGB
+`LEVIR-CD+`_,CD,Google Earth,-,985,2,"1,024x1,024",0.5,RGB
+`LoveDA`_,S,Google Earth,"CC-BY-NC-SA-4.0","5,987",7,"1,024x1,024",0.3,RGB
+`MapInWild`_,S,"Sentinel-1/2, ESA WorldCover, NOAA VIIRS DNB","CC-BY-4.0",1018,1,1920x1920,10--463.83,"SAR, MSI, 2020_Map, avg_rad"
+`Million-AID`_,C,Google Earth,-,1M,51--73,,0.5--153,RGB
+`NASA Marine Debris`_,OD,PlanetScope,"Apache-2.0",707,1,256x256,3,RGB
+`OSCD`_,CD,Sentinel-2,"CC-BY-4.0",24,2,"40--1,180",60,MSI
+`PASTIS`_,I,Sentinel-1/2,"CC-BY-4.0","2,433",19,128x128xT,10,MSI
+`PatternNet`_,C,Google Earth,-,"30,400",38,256x256,0.06--5,RGB
+`Potsdam`_,S,Aerial,-,38,6,"6,000x6,000",0.05,MSI
+`ReforesTree`_,"OD, R",Aerial,-,100,6,"4,000x4,000",0.02,RGB
+`RESISC45`_,C,Google Earth,-,"31,500",45,256x256,0.2--30,RGB
+`Rwanda Field Boundary`_,S,Planetscope,-,70,2,256x256,4.7,RGB + NIR
+`Seasonal Contrast`_,T,Sentinel-2,-,100K--1M,-,264x264,10,MSI
+`SeasoNet`_,S,Sentinel-2,"CC-BY-4.0","1,759,830",33,120x120,10,MSI
+`SEN12MS`_,S,"Sentinel-1/2, MODIS","CC-BY-4.0","180,662",33,256x256,10,"SAR, MSI"
+`SKIPP'D`_,R,"Fish-eye","CC-BY-4.0","363,375",-,64x64,-,RGB
+`So2Sat`_,C,Sentinel-1/2,"CC-BY-4.0","400,673",17,32x32,10,"SAR, MSI"
+`SpaceNet`_,I,WorldView-2/3 Planet Lab Dove,-,"1,889--28,728",2,102--900,0.5--4,MSI
+`SSL4EO`_-L,T,Landsat,"CC0-1.0",1M,-,264x264,30,MSI
+`SSL4EO`_-S12,T,Sentinel-1/2,"CC-BY-4.0",1M,-,264x264,10,"SAR, MSI"
+`SSL4EO-L Benchmark`_,S,Lansat & CDL,"CC0-1.0",25K,134,264x264,30,MSI
+`SSL4EO-L Benchmark`_,S,Lansat & NLCD,"CC0-1.0",25K,17,264x264,30,MSI
+`SustainBench Crop Yield`_,R,MODIS,"CC-BY-SA-4.0",11k,-,32x32,-,MSI
+`Tropical Cyclone`_,R,GOES 8--16,"CC-BY-4.0","108,110",-,256x256,4K--8K,MSI
+`UC Merced`_,C,USGS National Map,-,"2,100",21,256x256,0.3,RGB
+`USAVars`_,R,NAIP Aerial,"CC-BY-4.0",100K,-,-,4,"RGB, NIR"
+`Vaihingen`_,S,Aerial,-,33,6,"1,281--3,816",0.09,RGB
+`VHR-10`_,I,"Google Earth, Vaihingen",-,800,10,"358--1,728",0.08--2,RGB
+`Western USA Live Fuel Moisture`_,R,"Landsat8, Sentinel-1","CC-BY-NC-ND-4.0",2615,-,-,-,-
+`xView2`_,CD,Maxar,"CC-BY-NC-SA-4.0","3,732",4,"1,024x1,024",0.8,RGB
+`ZueriCrop`_,"I, T",Sentinel-2,-,116K,48,24x24,10,MSI
```

Comment:
Hi @adamjstewart, sorry for my late response - I missed your question. In case your question is still relevant I recommend to check with @0zgur0 who should know more about ZueriCrops as I also could not find if there's a license and what kind of